src/qt/forms/modaloverlay.ui

```diff
@@ -334,6 +354,33 @@ QLabel { color: rgb(40,40,40);  }</string>
             <number>10</number>
            </property>
            <item>
+            <widget class="QPushButton" name="pauseResumeVerification">
```

Comment:
I prefer to keep it because it's not pure download, it's the whole verification (and maybe we further extend what it does in the background later).